runtests.sh

```diff
@@ -37,9 +37,9 @@ export WORKER_ADMINISTRATION_REGION=api
 export SENTRY_DSN=''
 export PYTHONDONTWRITEBYTECODE=1
 export POSTGRESQL_USER='coreapi'
-export POSTGRESQL_PASSWORD='coreapi'
+export POSTGRESQL_PASSWORD='coreapipostgres'
 export POSTGRESQL_DATABASE='coreapi'
-export PGBOUNCER_SERVICE_HOST='coreapi-pgbouncer'
+export PGBOUNCER_SERVICE_HOST='coreapi-postgres'
```

Comment:
The problem here is that (as you already know) the host, where the script runs, can't resolve the `'coreapi-postgres'` hostname. Quick work-around could be to use `localhost` instead, since the `coreapi-postgres` container exposes the port (5432) to host. But in long term, I think the tests should run in a container which would be linked to the `coreapi-postgres` container - see [worker's Dockerfile.tests](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.tests) & [worker's runtest.sh](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/runtest.sh) for how it's done in worker.

Reply:
Thanks @jpopelka. This is really helpful.